local_tests.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/sh
+
+# Copyright (c) Outscale SAS
+#
+# SPDX-License-Identifier: BSD-3-Clause
+
+export OSC_PASSWORD='ashita wa dochida'
+export OSC_LOGIN=joe
+export OSC_USERNAME=joe
+
+export OMI_ID="ami-90067666"
+
+export OSC_SECRET_KEY=0000001111112222223333334444445555555666
+export OSC_ACCESS_KEY=11112211111110000000
+
+export OSC_ENDPOINT_API_PROTO="http://localhost:3000/api/v1"
+export OSC_ENDPOINT_API_NOPROTO="localhost:3000/api/v1"
+export OSC_ENDPOINT_API="localhost:3000/api/v1"
+export OSC_REGION="vp-ware-3"
+export OSC_PROTOCOL="http"
+
+export OSC_USING_RICOCHET="oui"
+
+if [ "$#" -eq 0 ]; then
+
+    if [ ! -d "osc-ricochet-2" ]; then
+	git clone https://github.com/outscale-mgo/osc-ricochet-2
+    fi
```

Comment:
Are you going to upgrade to the latest stable version once it's available instead of having the latest version which may not be stable ?

Reply:
yes